### PR TITLE
Add todo-app example

### PR DESCRIPTION
**Pull request by andresmgot**
This PR adds as an example a TODO list app.

Check the READMEs for instructions of how to deploy it.

### CODE REVIEW

File: examples/todo-app/frontend/react-redux/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ENV NODE_ENV=production
+
+RUN npm install yarn --global
+
+COPY . /app
+
+RUN yarn install
+RUN npm rebuild node-sass
+RUN yarn run build
```
Comment (tompizmor) [inline examples/todo-app/frontend/react-redux/Dockerfile]: Building the image I get this error:

```
Step 8/11 : RUN yarn run build
 ---> Running in 1740e4d2f3a0
yarn run v0.27.5
$ webpack
sh: 1: webpack: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run build' returned a non-zero code: 1
```

File: examples/todo-app/frontend/react-redux/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# React-Redux frontend
+
+This is a frontend for our `todo` application which is implemented with the help of [React](http://reactjs.org) and [Redux](http://reduxjs.org).
+
+Do the following to setup and use the frontend
+
+1. Make sure that you've deployed the backend of the `todo` application
+2. Run `npm install` to install the used npm packages
+3. Go to `app/js/actions/index.js` and update the `API_URL` with the endpoint of your deployed `todo` Serverless service (e.g. `https://somestring.execute-api.us-east-1.amazonaws.com/dev/todos`)
+3. Go to `app/js/actions/index.js` and update the `API_URL` with the endpoint of your deployed `todo` Serverless service (e.g. `https://192.168.99.100`)
```
Comment (andresmgot) [inline examples/todo-app/frontend/react-redux/README.md]: Sorry the docker file is not needed. I have deleted it.

File: examples/todo-app/frontend/react-redux/Dockerfile (hunk 1 above)
Comment (andresmgot) [inline examples/todo-app/frontend/react-redux/Dockerfile]: _removed this image_

File: examples/todo-app/frontend/react-redux/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# React-Redux frontend
+
+This is a frontend for our `todo` application which is implemented with the help of [React](http://reactjs.org) and [Redux](http://reduxjs.org).
+
+Do the following to setup and use the frontend
+
+1. Make sure that you've deployed the backend of the `todo` application
+2. Run `npm install` to install the used npm packages
+3. Go to `app/js/actions/index.js` and update the `API_URL` with the endpoint of your deployed `todo` Serverless service (e.g. `https://192.168.99.100`)
+4. Open up a browser on [localhost:8080](http://localhost:8080) and play around with the application
```
Comment (tompizmor) [inline examples/todo-app/frontend/react-redux/README.md]: add the step 5 to run `npm start` to start the frontend

**Comment by tompizmor** [comment]
LGTM! 👍

**Pull request merged**